util/mkrc.pl

```diff
@@ -25,7 +25,7 @@
     $vft = "VFT_APP";
 }
 
-my $YEAR = [localtime()]->[5] + 1900;
+my $YEAR = [localtime($ENV{SOURCE_DATE_EPOCH} || time())]->[5] + 1900;
```

Comment:
Doesn't localtime() depend on the ... local time zone?  I.e., is not actually reproducible for epoch values near a year transition, depending on the time zone currently in use?

Reply:
you're probably right but who cares. ;)

Reply:
Then what about just replacing localtime() with gmtime()? Not that it matters much, just for a strict correctness.

Reply:
I updated the commit to use gmtime() instead of localtime().